src/controller/python/chip/utils/DeviceProxyUtils.cpp

```diff
@@ -59,4 +59,23 @@ uint32_t pychip_DeviceProxy_ComputeRoundTripTimeout(DeviceProxy * device, uint32
         ->ComputeRoundTripTimeout(System::Clock::Milliseconds32(upperLayerProcessingTimeoutMs))
         .count();
 }
+
+/**
+ * @brief
+ *
+ * This gets the MaxPathsPerInvoke supported by remote node.
+ *
+ * A valid DeviceProxy pointer with a valid, established session is required for this method.
+ */
+uint16_t pychip_DeviceProxy_GetMaxPathsPerInvoke(DeviceProxy * device, uint32_t upperLayerProcessingTimeoutMs)
+{
+    VerifyOrDie(device != nullptr);
+
+    auto * deviceProxy = static_cast<DeviceProxy *>(device);
+    VerifyOrDie(deviceProxy->GetSecureSession().HasValue());
```

Comment:
We can't do VerifyOIrDie here --> this may have happened asynchronously in between GetConnectedDeviceSync and the self._ChipStack.Call due to comms errors. Suggest instead to return 0 on error (which is not a valid value).